go/host/rpc/clientapi/client_api_eth.go

```diff
@@ -38,17 +38,17 @@ func (api *EthereumAPI) ChainId() (*hexutil.Big, error) { //nolint:stylecheck,re
 	return (*hexutil.Big)(big.NewInt(api.host.Config().ObscuroChainID)), nil
 }
 
-// BlockNumber returns the height of the current head rollup.
-// # TODO - #718 - Switch to returning height based on current batch.
+// BlockNumber returns the height of the current head batch.
 func (api *EthereumAPI) BlockNumber() hexutil.Uint64 {
-	header, err := api.host.DB().GetHeadRollupHeader()
+	header, err := api.host.DB().GetHeadBatchHeader()
```

Comment:
I switched this method over to retrieving the number based on the batch, not the rollup. This ensures that the sims are exercising the batch distribution code. In a subsequent PR, I will move other APIs over to consult the batches, rather than the rollups, as per the new design.